lib/asset_remover.rb

```diff
@@ -1,4 +1,39 @@
 class AssetRemover
+  def remove_attachment_file
+    target_dir = File.join(Whitehall.clean_uploads_root, 'system', 'uploads', 'attachment', 'file')
```

Comment:
Just to double-check: This is distinct from the attachments which are still managed by the app, right? They've got `attachment_data` in the path, I think.

Reply:
Yes, that's right. This directory `system/uploads/attachment/file` has 6882 files in it and hasn't had any new files added to it since at least mid-December last year. It looks like the mounted uploader was removed from `Attachment` in dde83ebde0.